netebpfext/net_ebpf_ext_program_info.h

```diff
@@ -71,6 +76,12 @@ static const ebpf_program_section_info_t _ebpf_bind_section_info[] = {
      BPF_PROG_TYPE_BIND,
      BPF_ATTACH_TYPE_BIND}};
 
+enum _sock_addr_helper_functions
+{
+    SOCK_ADDR_HELPER_GET_CURRENT_PID_TGID,
+    SOCK_ADDR_HELPER_SET_REDIRECT_CONTEXT,
+};
```

Comment:
nit: it looks like these enums are only used in test / mock code. Should these enums be then moved to some test headers?

Reply:
The reason why I kept it in the same file as the helper function prototype, so that when the list of helpers are changed, this enum gets updated as well.